docs/modules/plugin/pages/introduction.adoc

```diff
@@ -0,0 +1,58 @@
+= Asciidoctor Maven Plugin
+
+The Asciidoctor Maven Plugin is used to convert AsciiDoc documents using https://asciidoctor.org[Asciidoctor].
+The plugin offers a comprehensive way to configure a conversion process using {uri-asciidoctorj}[Asciidoctorj] in a declarative way.
+
+== Goals overview
+
+The plugin has 3 goals.
+None of these are bound to any specific phase.
+
+* xref:goals/process-asciidoc.adoc[asciidoctor:process-asciidoc]:
```

Comment:
I think this would look better as a dlist:

```
xref:goals/process-asciidoc.adoc[asciidoctor:process-asciidoc]::
main goal of the plugin, it is used to convert documents during a normal Maven build.
```

Reply:
Sure no problem.
On the topic, truth I am never sure when to use them. This is one of those things that I would appreciate to have in a style guide.